render/detailed/summary.go

```diff
@@ -53,33 +64,58 @@ func MakeColumn(id string) Column {
 type NodeSummary struct {
 	ID           string        `json:"id"`
 	Label        string        `json:"label"`
-	Linkable     bool          `json:"linkable"` // Whether this node can be linked-to
+	LabelMinor   string        `json:"label_minor"`
+	Rank         string        `json:"rank"`
+	Shape        string        `json:"shape,omitempty"`
+	Stack        bool          `json:"stack,omitempty"`
+	Linkable     bool          `json:"linkable,omitempty"` // Whether this node can be linked-to
+	Pseudo       bool          `json:"pseudo,omitempty"`
 	Metadata     []MetadataRow `json:"metadata,omitempty"`
 	DockerLabels []MetadataRow `json:"docker_labels,omitempty"`
```

Comment:
It seems a shame a side effect of this change is that everything now has a docker labels.

I feel like we should change `Metadata` to be a `map[table name][]MetadataRow` to deal with this case.  Punt this for now though - maybe file a techdebt issue?